website/pages/en/querying/querying-best-practices.mdx

```diff
@@ -65,8 +65,6 @@ Failing to follow the above rules will end with an error from the Graph API.
 
 For a complete list of rules with code examples, please look at our GraphQL Validations guide.
 
-<br />
```

Comment:
The extra spacing looks weird IMO; I think we should be consistent with spacing.